arangod/Graph/Enumerators/WeightedTwoSidedEnumerator.cpp

```diff
@@ -141,21 +145,21 @@ auto WeightedTwoSidedEnumerator<QueueType, PathStoreType, ProviderType,
 
 template<class QueueType, class PathStoreType, class ProviderType,
          class PathValidator>
-auto WeightedTwoSidedEnumerator<QueueType, PathStoreType, ProviderType,
-                                PathValidator>::Ball::fetchResult(Candidate&
-                                                                      candidate)
-    -> void {
+auto WeightedTwoSidedEnumerator<
+    QueueType, PathStoreType, ProviderType,
+    PathValidator>::Ball::fetchResult(CalculatedCandidate& candidate) -> void {
   // TODO: Right now almost a copy of method below - optimize this.
   std::vector<Step*> looseEnds{};
+  auto& [weight, first, second] = candidate;
```

Comment:
What is the meaning of `first` and `second` here? is this the source and the sink of an edge? if so you could give this meaningful names here and below not assign to `step` but use the varaible directly. That'd make the code more concise.

Reply:
done